tests/conformance/scripts/generate-inputs.ts

```diff
@@ -82,7 +82,7 @@ function generateFile(features: Set<TestFeatures>, records: TestDataRecord[]) {
         if (features.has(TestFeatures.UseAttachmentIndex)) {
           attachmentIndexes.push({
             name: record.name,
-            length,
+            length: length - (features.has(TestFeatures.AddExtraDataToRecords) ? 3n : 0n),
```

Comment:
I don't understand why we are subtracting from the length here. The length is the length of the attachment record (including padding).